controllers/chat.js

```diff
@@ -0,0 +1,19 @@
+const chatService = require('~/services/chat')
+
+const createChat = async (req, res) => {
+  const { id: author, role: authorRole } = req.user
+  const { chatMember, chatMemberRole } = req.body
+
+  const chat = await chatService.createChat({
```

Comment:
maybe it would be better to move this combining of data to service and just pass to object with author data and the second one with chat member data

Reply:
Fixed.